### DISCUSSION TITLE
Why do some colors have 8 hex digits?

### DISCUSSION THREAD

**Discussion by jsonnentag** [rrirower/DP2SceneGraphWizard#80] [Q&A]
In testing out the latest (Beta 2) I was going through changing colors again (since the fonts settings don't modify the Play/Episodes/... text) and noticed in multiple places that some color values have 8 hex digits rather than just the 6 (2 each for red, green blue).  I am GUESSING that at some point you did a search and replace and some digits were left or something.  (???)

Anyway, it appears that they are ignored when processed.  As far as I can tell they are anyway.

Here are the locations I have found:

fontsupport.brs (down at the end for the default font)

EpisodesListItemComponent.xml  (title, info, and description settings and I think rectangle color has extras too)

**Comment by jsonnentag** [comment]
This is somewhat different, but a question about the fonts. . . .

Do you know if the font size setting has ZERO effect if the font is set to a system one (such at those listed here).

[https://developer.roku.com/docs/references/scenegraph/typographic-nodes/font.md](url)

    SmallestSystemFont
    SmallestBoldSystemFont
    SmallSystemFont
    SmallBoldSystemFont
    MediumSystemFont
    MediumBoldSystemFont
    LargeSystemFont
    LargeBoldSystemFont

I got rid of all the fonts except for the dot used in the videos extras info line and just put the system ones to save file space and since I'm OK with the system ones (well, and I don't like the italics one because it is too hard to read).

What is the massive sized (859K) NotoEmoji-Regular.ttf font used for?

**Reply by rrirower** [reply]
- I think the difference is in the use of the Alpha bit in the color scheme.  I used a hex color table to select colors and since it works, I'll assume that BrightScript ignores that bit.
- That may be a problem since the affected fields are defined as **MultiStyleLabe**l and there's no setting for size on them.  It uses the attached font for the size.
- You can remove any font you don't want.  They were there for testing and to provide users with options.
